index.js

```diff
@@ -175,6 +190,11 @@ try {
   }
 }
 
+
+if (cypressConfig.testFiles === undefined) {
+  cypressConfig.testFiles = argv.testFiles;
+}
```

Comment:
We should probably default to `cypressConfig.testFiles` and override the value when `argv.testFiles` is provided.

Reply:
Isn't that what's happening here? The only time we set the config to `testFiles` is when there isn't a value set in the cypressConfig. Maybe I'm confused

Reply:
My thought here was that we should make usage as easy as possible by always defaulting to values found in the users `Cypress.json` file, while also allowing users to set an overridden value using the command line flags.

For example, here it might say `const testFiles = argv.testFiles || cypressConfig.testFiles;`

Reply:
Sorry could you give me an example of what you mean :/ I'm a little confused